### ISSUE TITLE
Cannot add additional accounts using talkback

### ISSUE THREAD

**Issue by jsoref** [tuskyapp/Tusky#4220]
The row containing the active profile and the disclosure widget below the list of accounts isn't reachable with TalkBack:
![image](https://github.com/tuskyapp/Tusky/assets/2119212/6689d5b7-7783-4a7a-9cb3-6dc09b025a04)

You can select the various profile items:
![image](https://github.com/tuskyapp/Tusky/assets/2119212/be36df6b-b8dd-4c74-9f38-7bc50ca1b50a)
But the next item reachable after profiles via TalkBack is:
![image](https://github.com/tuskyapp/Tusky/assets/2119212/d62b3161-99a6-4168-94d3-6512566a8bdf)

Unfortunately, Add account is hidden behind that skipped row.

Reported: https://mastodon.social/@jsoref_using_a_screen_reader/111717811996531862
* * * *
- Tusky Version: 24.1
- Android Version: 14
- Android Device: Google Pixel 6a
- Mastodon instance (if applicable):

- [x] I searched or browsed the repo’s other issues to ensure this is not a duplicate.
